earth_enterprise/src/server/wsgi/serve/publish/publish_manager_helper.py

```diff
@@ -1417,9 +1417,11 @@ def _GetVhNameToBaseUrlDict(self):
       (scheme://host[:port]) -  {vh_name: vh_base_url}
     """
     vh_list = self.QueryVhList()
-    vhname_to_baseurl_dct = {
-        vh_name: self.GetVhBaseUrl(vh_url, vh_ssl) for (
-            vh_name, vh_url, vh_ssl) in vh_list}
+    vhname_to_baseurl_dct = dict(
```

Comment:
:stuck_out_tongue: You don't need a for loop to create an empty dictionary, but it will work.

It could be just:

```Python
    vh_list = self.QueryVhList()
    vhname_to_baseurl_dct = {}
  
    for (vh_name, vh_url, vh_ssl) in vh_list:
      vhname_to_baseurl_dct[vh_name] = self.GetVhBaseUrl(vh_url, vh_ssl)
    return vhname_to_baseurl_dct
```

Reply:
vhname_to_baseurl_dct = {}   throws error in 2.6, hence this hack

Reply:
I don't have a problem with it with Python 2.6.6.

Reply:
You could also use `vhname_to_baseurl_dct = dict()`.

Reply:
for first suggestion , as I mentioned this is the error:
>>> import p
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "p.py", line 22
    vhname_to_baseurl_dct = dict{}
                                ^
SyntaxError: invalid syntax

Reply:
Since its approved, I am not planning to make any more changes although seemingly vhname_to_baseurl_dct = dict() this is not throwing error, but not tested, so not changing it.